.github/workflows/python-package.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9, 2.7]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Don't you test py 3.10 & 3.11 (and 3.12 is coming)?

Reply:
I mean to have this PR contain minimal changes needed for dropping Python 2.7, so that your review will not be distracted.

There is already another pending PR that is to add py 3.10, 3.11, 3.12 testing.